lib/logmsg/tests/test_tags.c

```diff
@@ -82,34 +71,33 @@ test_tags(void)
       name = get_tag_by_id(i);
 
       tag_name = log_tags_get_by_id(i);
-      test_msg("Looking up tag by id %d %s(%s)\n", i, tag_name, name);
+      cr_log_info("Looking up tag by id %d %s(%s)\n", i, tag_name, name);
       if (tag_name)
         {
           if (!g_str_equal(tag_name, name))
-            test_fail("Bad tag name for id %d %s (%s)\n", i, tag_name, name);
+            cr_assert_eq(name, tag_name, "Bad tag name for id %d %s (%s)\n", i, tag_name, name);
```

Comment:
Strings can't be compared with `cr_assert_eq`, `cr_assert_str_eq` should be used instead. The condition (`if (!g_str_equal(tag_name, name))`) is unnecessary if you use the assertion API.

Reply:
Ok. I will change that. Thanks.

Reply:
@MrAnno how to solve the make style-check error?